diff --git a/peppy/project.py b/peppy/project.py
index e1f187d..3e27b38 100644
--- a/peppy/project.py
+++ b/peppy/project.py
@@ -2,7 +2,7 @@
 Build a Project object.
 """
 import os
-from collections import Mapping
+from collections.abc import Mapping
 from logging import getLogger
 
 import pandas as pd
diff --git a/peppy/sample.py b/peppy/sample.py
index 590df56..683727a 100644
--- a/peppy/sample.py
+++ b/peppy/sample.py
@@ -106,8 +106,6 @@ class Sample(PathExAttMap):
             :param str name: name of the object to represent.
             :param Iterable[str] to_skip: names of attributes to ignore.
             """
-            from collections import Mapping
-
             from attmap import AttMap
             from pandas import Series, isnull
 
